ivre/db/__init__.py

```diff
@@ -1215,24 +1215,25 @@ def parse_args(self, args, flt=None):
         flt = super(DBView, self).parse_args(args, flt=flt)
         if args.ssl_ja3_client is not None:
             flt = self.flt_and(flt, self.searchja3client(
-                value_or_hash=args.ssl_ja3_client
+                value_or_hash=(False if args.ssl_ja3_client is False else
+                               utils.str2regexp(args.ssl_ja3_client))
             ))
         if args.ssl_ja3_server is not None:
             if args.ssl_ja3_server is False:
                 # There are no additional arguments
                 flt = self.flt_and(flt, self.searchja3server())
             else:
-                splitted = args.ssl_ja3_server.split(':', 1)
-                if len(splitted) == 1:
+                split = args.ssl_ja3_server.split(':', 1)
```

Comment:
Maybe you could apply `utils.str2regexp` on every elements of the array instead of manually line 1230, 1235 and 1236.